packages/node_modules/@node-red/editor-client/src/js/ui/subflow.js

```diff
@@ -508,10 +508,8 @@ RED.subflow = (function() {
         var activeSubflow = RED.nodes.subflow(id);
 
         RED.nodes.eachNode(function(n) {
-            if (!keepInstanceNodes && n.type == "subflow:"+id) {
-                removedNodes.push(n);
-            }
-            if (n.z == id) {
+            if (n.z === id || (!keepInstanceNodes && n.type === "subflow:" + id)) {
```

Comment:
Should be handled in `RED.nodes.remove`
```suggestion
            if (!keepInstanceNodes && n.type == "subflow:"+id) {
                removedNodes.push(n);
            }
            if (n.z == id) {
```

Reply:
Can you clarify if you are proposing a change to your own PR for me to review? Or something else?

Reply:
Exactly, I made a change in this file but after looking at other elements I come to the conclusion that this change should not take place here. So you can commit this revert.